src/test/unit/test_lumigo_utils.py

```diff
@@ -465,12 +465,12 @@ def test_report_json_extension_spans_mode(monkeypatch, reporter_mock):
                 i: "a" * size_factor,
             }
         )
-    report_json(None, spans)
+    report_json(region=None, msgs=spans, is_start_span=False)
 
     files_paths = []
     for span in spans:
         files_paths.append(get_span_file_name(span, "span"))
-    done_object = {"spansCount": len(spans)}
+    done_object = {"spansCount": len(spans) + 1}
```

Comment:
if `Configuration.send_only_if_error` then you will not send `+1` spans (because we're not sending the start span)

Reply:
please add test to this scenario :)

Reply:
Is there some way to know how manny spans in total exists?
i know in the node its there

Reply:
nothing that I'm aware of 🤔